_pages/dev/features/scroll-to-top.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Scroll to Top
+feature:
+  - name: Scroll to Top
+    spa_version: 5.0
+    cx_version: 2105
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 5.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The scroll to top feature allows users to quickly scroll back to the top of the page.
+
+---
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+---
+
+## Enabling Scroll to Top
+
+Scroll to top is part of the `@spartacus/storefront` library, and installed during the initial setup. To enable this feature in your storefront add the scroll to top CMS data to a content slot using ImpEx. 
```

Comment:
Suggestion:

To enable this feature in your storefront, add the Scroll to Top CMS Component to a content slot in the web site pages you wish to use the feature with.  To add Scroll to Top to the whole web site, you can add the Scroll to Top component in a shared content slot, like the footer.